### PR TITLE
feat: add 0518 space factory deployment script

### PR DESCRIPTION
**Pull request by jparklev**
## Motivation

Update the Space Factory with the [1.3 fixes](https://github.com/sense-finance/space-v1/pull/25) and sync the pools registry

## Solution

Write a script to deploy the Space Factory, then collect the pool addresses from the previous Space Factory and pass them to the newly deployed Factory's registry.

Dependent on https://github.com/sense-finance/space-v1/pull/27

## Implementer Checklist

N/A

### CODE REVIEW

File: pkg/deployments/tasks/20220518-space-factory/index.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+const { mainnet } = require("./input");
+
+const { SENSE_MULTISIG, BALANCER_VAULT } = require("../../hardhat.addresses");
+
+const dividerAbi = require("./abi/Divider.json");
+const oldSpaceFactoryAbi = require("./abi/OldSpaceFactory.json");
+const spaceFactoryAbi = require("./abi/SpaceFactory.json");
+const peripheryAbi = require("./abi/Periphery.json");
+
+task("20220518-space-factory", "Deploys long term wstETH adapter").setAction(async ({}, { ethers }) => {
```
Comment (fedealconada) [inline pkg/deployments/tasks/20220518-space-factory/index.js]: Change description?

**Review by fedealconada — APPROVED** [review]
lgtm!

File: pkg/deployments/tasks/20220518-space-factory/index.js (hunk 1 above)
Comment (jparklev) [reply pkg/deployments/tasks/20220518-space-factory/index.js]: fixed https://github.com/sense-finance/sense-v1/pull/248/commits/3b93b991f5e75ee0869211abf23a3058cf72bc68

**Review by fedealconada — APPROVED** [review]
LGTM!

**Pull request merged**